src/v/redpanda/admin_server.cc

```diff
@@ -263,10 +264,20 @@ get_boolean_query_param(const ss::httpd::request& req, std::string_view name) {
 }
 
 void admin_server::configure_metrics_route() {
-    ss::prometheus::config metrics_conf;
-    metrics_conf.metric_help = "redpanda metrics";
-    metrics_conf.prefix = "vectorized";
-    ss::prometheus::add_prometheus_routes(_server, metrics_conf).get();
+    ss::prometheus::add_prometheus_routes(
+      _server,
+      {.metric_help = "redpanda metrics",
+       .prefix = "vectorized",
+       .handle = ss::metrics::impl::default_handle(),
+       .route = "/metrics"})
+      .get();
+    ss::prometheus::add_prometheus_routes(
+      _server,
+      {.metric_help = "redpanda metrics",
+       .prefix = "redpanda",
+       .handle = ssx::public_metrics_handle,
+       .route = "/public_metrics"})
```

Comment:
I'm not sure public conveys meaning outside of the cloud. Naming is hard.

Reply:
@BenPope i think i meant it in that /metrics become internal to the dev team, nothing related to cloud. this is for public consumption - i.e.: stable api semantics.

Reply:
Oh, I see. I don't have a strong opinion on it, happy with "public".